spec/services/wallet_transactions/create_service_spec.rb

```diff
@@ -22,7 +22,8 @@
       {
         name: 'New Wallet',
         wallet_id: wallet.id,
-        customer_id: customer.id,
+        customer_id: customer.customer_id,
+        organization_id: organization.id,
```

Comment:
Can we add some tests concerning the changes about `granted_credits`?

Reply:
Sorry for that, specs are added in the next PR (since this create service is not finished here - paid credits handling is missing here)